[2025-01-19T20:37:32.448+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_fill_dm_account_turnover_f scheduled__2025-01-18T00:00:00+00:00 [queued]>
[2025-01-19T20:37:32.463+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_fill_dm_account_turnover_f scheduled__2025-01-18T00:00:00+00:00 [queued]>
[2025-01-19T20:37:32.464+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-19T20:37:32.482+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_fill_dm_account_turnover_f> on 2025-01-18 00:00:00+00:00
[2025-01-19T20:37:32.487+0000] {standard_task_runner.py:60} INFO - Started process 180 to run task
[2025-01-19T20:37:32.491+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_fill_dm_account_turnover_f', 'scheduled__2025-01-18T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpbvag_vhh']
[2025-01-19T20:37:32.493+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask call_fill_dm_account_turnover_f
[2025-01-19T20:37:33.555+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_fill_dm_account_turnover_f scheduled__2025-01-18T00:00:00+00:00 [running]> on host b8f36627e35c
[2025-01-19T20:37:33.635+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dmitry' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_fill_dm_account_turnover_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-18T00:00:00+00:00'
[2025-01-19T20:37:33.636+0000] {sql.py:276} INFO - Executing: DO $$
DECLARE
    day DATE := '2018-01-01';
BEGIN
    WHILE day <= '2018-01-31' LOOP
        CALL ds.fill_account_turnover_f(day);
        day := day + INTERVAL '1 day';
    END LOOP;
END $$;
[2025-01-19T20:37:33.647+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-19T20:37:33.664+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-19T20:37:33.731+0000] {sql.py:450} INFO - Running statement: DO $$
DECLARE
    day DATE := '2018-01-01';
BEGIN
    WHILE day <= '2018-01-31' LOOP
        CALL ds.fill_account_turnover_f(day);
        day := day + INTERVAL '1 day';
    END LOOP;
END $$;, parameters: None
[2025-01-19T20:37:34.456+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_fill_dm_account_turnover_f, execution_date=20250118T000000, start_date=20250119T203732, end_date=20250119T203734
[2025-01-19T20:37:34.514+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-19T20:37:34.691+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
